components/integrations_marketplace/marketplace_modal.jsx

```diff
@@ -0,0 +1,185 @@
+                                        id='marketplace_modal.install_plugins'
+                                        defaultMessage='Install Plugins'
+                                    />
+                                </a>
+                            ) : null
+                            }
+                        </div>
+                    )
+            }
+        </div>);
+    }
+
+    render() {
+        //WIP: To add pagination section
+        const input = (
+            <div className='filter-row filter-row--full'>
+                <div className='col-sm-12'>
+                    <QuickInput
+                        id='searchChannelsTextbox'
+                        ref='filter'
+                        className='form-control filter-textbox search_input'
+                        placeholder={{id: t('marketplace_modal.search'), defaultMessage: 'Search Plugins'}}
+                        inputComponent={LocalizedInput}
+                        onInput={this.doSearch}
+                    />
+                </div>
+            </div>
+        );
+
+        return (
+            <RootPortal>
+                <FullScreenModal
+                    show={Boolean(this.props.show)}
+                    onClose={this.close}
+                >
+                    <div className='MarketplaceModal'>
+                        <h1>
+                            <FormattedMessage
+                                id='marketplace_modal.title'
+                                defaultMessage='Plugins Marketplace'
+                            />
+                        </h1>
+                        {input}
+                        <Tabs
+                            id='marketplaceTabs'
+                            className='tabs'
+                            defaultActiveKey='allPlugins'
+                            activeKey={this.state.tabKey}
+                            onSelect={this.handleSelect}
+                            unmountOnExit={true}
+                        >
+                            <Tab
+                                eventKey={MarketplaceTabs.ALL_PLUGINS}
+                                title={localizeMessage('marketplace_modal.tabs.all_plugins', 'All Plugins')}
+                            >
+                                {this.state.loading ?
+                                    <LoadingScreen/> : (
+                                        this.getPluginsListContent(this.props.marketplacePlugins, false)
+                                    ) }
+                            </Tab>
+                            <Tab
+                                eventKey={MarketplaceTabs.INSTALLED_PLUGINS}
+                                title={localizeMessage('marketplace_modal.tabs.installed_plugins', `Installed (${this.props.installedPlugins.length})`)}
+                            >
+                                {this.getPluginsListContent(this.props.installedPlugins, false)}
```

Comment:
shouldn't be here the second parameter `true`?

Reply:
Yes! 🤦‍♀️Good catch!